virttest/virt_vm.py

```diff
@@ -464,6 +464,30 @@ def __init__(self, model=None, vendor=None, flags=None, family=None,
         self.threads = threads
 
 
+def session_handler(func):
+    """
+    decorator method to handle uri and session for libvirt
+    """
+    def manage_session(*args, **kwargs):
```

Comment:
better to define like `def manage_session(vm, *args, **kwargs)` and wrap it with `functools.wraps`.